package.json

```diff
@@ -144,10 +144,12 @@
     ]
   },
   "dependencies": {
+    "@tinymce/tinymce-react": "^2.4.0",
```

Comment:
dependencies should be listed in alphabetical order. move lower where 'T' is supposed to be.